src/components/view-center.js

```diff
@@ -0,0 +1,48 @@
+import { x } from '@xstyled/styled-components'
+import BancoComida from '../iamges/banco-de-comida.jpeg'
+import { useNavigate } from 'react-router-dom';
+
+export default function ViewCenter() {
```

Comment:
Change the name of this component to CenterCard

```suggestion
export default function CenterCard() {
```